### PR TITLE
Add realy project for mobile notifications

### PR DESCRIPTION
**Pull request by ManfredKarrer**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -18,7 +18,8 @@
     <mapping directory="$PROJECT_DIR$/monitor" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/p2p" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/pricenode" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/seednode" vcs="Git" />
+	<mapping directory="$PROJECT_DIR$/seednode" vcs="Git" />
```
Comment (ripcurlx) [inline .idea/vcs.xml]: too many spaces

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```
Comment (ripcurlx) [inline .idea/codeStyles/codeStyleConfig.xml]: Is there a specific reason for this change?

Comment (ManfredKarrer) [reply .idea/codeStyles/codeStyleConfig.xml]: Upps was a mistake. My IntelliJ always rewrites that from some local settings... Will revert

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**